.env

```diff
@@ -12,5 +14,14 @@ UPSTREAM_DNS=8.8.8.8
 ## Storage path for the cached data
 CACHE_ROOT=/lancache
 
-## Which type of steam caching to use [generic|steamcache]
-STEAMCACHE_CONTAINER=generic
+## Change this to customise the size of the memory cache (default 500m)
+CACHE_MEM_SIZE=500m
+
+## Change this to customise the size of the disk cache (default 1000000m)
+## If you have more storage, you'll likely want to increase this
+## The cache server will prune content on a least-recently-used basis if it
+## starts approaching this limit
+CACHE_DISK_SIZE=1000000m
```

Comment:
i'd probably move this to the top of this lower section as it's the one people are most likely to use, or set it to CHANGEME?

Reply:
Setting it to `CHANGEME` would cause this to break at startup, which I'd prefer to avoid. Ideally this repo should serve as a "clone and run" example of how to get a cache instance going.

Reply:
The .env needs customising anyway! Didn't really think about that